com.unity.shadergraph/Editor/Data/Legacy/AbstractMaterialNode0.cs

```diff
@@ -15,6 +15,8 @@ class AbstractMaterialNode0
 
         public string m_KeywordGuidSerialized;
 
+        public string m_DropdownGuidSerialized;
```

Comment:
Why does this need to be added to the legacy struct?  This should only be for upgrading from old graphs, which should never have the dropdown...

Reply:
I've removed this one.